### PR TITLE
#27 - add language column

### PR DESCRIPTION
**Pull request by AdamJWise**
- Added language column to the table, to display the main programming language as detected by github
- For now, using inline style to compensate for issue in main.css generation, but also added style in main.scss

### CODE REVIEW

File: server/components/project_repository.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ProjectRepository {
 
   getProjects(user, page = 0) {
     const client = github.createClient(user);
-    return client.repos.getReposAsync({
+    return client.repos.gmn etReposAsync({
```
Comment (stopyoukid) [inline server/components/project_repository.js]: Typo?

Comment (AdamJWise) [inline server/components/project_repository.js]: yes, weird typo after I had verified things.. I've updated the branch with the fix.

**Comment by darthtrevino** [comment]
:shipit: @AdamJWise Can you take a look at the CSS issue now? I'd like to kill the duplicate styling in the React components.

**Comment by AdamJWise** [comment]
Sure, I'll take a look now at main.css - Should I click 'Merge pull request' on each of these three pull requests? Thanks.

**Comment by stopyoukid** [comment]
@AdamJWise Yes, we try to follow github flow: http://labs.atsid.com/engineering/github/#flow .  But, basically, once you get 1 or more :shipit:, then you merge the pull request.

**Pull request merged**